webpack/scenes/Subscriptions/Manifest/ManageManifestModal.js

```diff
@@ -150,166 +147,145 @@ class ManageManifestModal extends Component {
-                            secondaryContent={
-                              // eslint-disable-next-line react/no-danger
-                              <p dangerouslySetInnerHTML={{
-                              __html: DeleteManifestModalText,
-                              }}
-                              />
-                            }
-                            primaryActionButtonContent={__('Delete')}
-                            primaryActionButtonBsStyle="danger"
-                            primaryAction={() => this.deleteManifest()}
-                            secondaryActionButtonContent={__('Cancel')}
-                            secondaryAction={() => this.showDeleteManifestModal(false)}
-                            onHide={() => this.showDeleteManifestModal(false)}
-                            accessibleName="deleteConfirmationDialog"
-                            accessibleDescription="deleteConfirmationDialogContent"
-                          />
-                        </Col>
-                      </FormGroup>
-                    </React.Fragment>
-                  }
-                </Form>
-              </Tab>
-            }
-            <Tab eventKey={2} title={__('Manifest History')}>
-              <LoadingState loading={manifestHistory.loading} loadingText={__('Loading')}>
-                <Table
-                  rows={manifestHistory.results}
-                  columns={columns}
-                  emptyState={emptyStateData()}
-                />
-              </LoadingState>
+                          {canDeleteManifest &&
+                          <React.Fragment>
+                            <TooltipButton
+                              renderedButton={(
+                                <Button
+                                  disabled={!manifestExists(organization) || actionInProgress}
+                                  bsStyle="danger"
+                                  onClick={this.showDeleteManifestModal}
+                                >
+                                  {__('Delete')}
+                                </Button>
+                                )}
+
+                              tooltipId="delete-manifest-button-tooltip"
+                              tooltipText={this.disabledTooltipText()}
+                              tooltipPlacement="top"
+
+                            />
+                          </React.Fragment>
+                          }
+                        </div>
+                        <ForemanModal title={__('Confirm delete manifest')} id={DELETE_MANIFEST_MODAL_ID}>
+                          <DeleteManifestModalText />
+                          <ForemanModal.Footer>
+                            <Button bsStyle="default" onClick={this.hideDeleteManifestModal}>
+                              {__('Cancel')}
+                            </Button>
+                            <Button bsStyle="danger" onClick={() => this.deleteManifest()}>
```

Comment:
Can this be ` onClick= {this.deleteManifest}` ?

Reply:
@MariaAga updated